linuxdoc/rstKernelDoc.py

```diff
@@ -63,6 +66,37 @@
         EXPORT_SYMBOL macro either in ``<src-filename>`` or in any of the files
         specified by ``<src-fname-pattern>``.
 
+    ``exp-method <method>``
+        Change the way exported symbols are specified in source code.
+        Default value ('macro') if not provided can be set globally by
+        kernel_doc_exp_method in the sphinx configuration.
+	``<method>`` must one of the following value:
+
+        ``macro``
+            Exported symbols are specified by macros (whose names are
+            controlled by ```exp-ids` option) invoked in the source the
+            following way: THIS_IS_AN_EXPORTED_SYMBOL(symbol)
+
+        ``attribute``
+            Exported symbols are specified definition using a specific
+            attribute (controlled by ```exp-ids` option) either in their
+            declaration or definition:
+            THIS_IS_AN_EXPORTED_SYMBOL int symbol(void* some_arg) {...}
+
+    ``exp-ids <identifier [, identifiers [, ...]]>``
+        Use the specified list of identifiers instead of default value:
+        EXPORT_SYMBOL, EXPORT_SYMBOL_GPL, EXPORT_SYMBOL_GPL_FUTURE. Default
+        value can be overriden globally by sphinx configuration option:
+        kernel_doc_exp_ids
+
+    ``known-attrs <attr [, attrs [, ...]]>``
+        Specified a list of function attribute that are known and must be
+        hidden when displaying function prototype. When ``exp-method`` is
+        set to 'attribute' the list in ``exp-ids`` is considered as known
+        and added implicitely to this list of known attributes. The default
+        list is empty and can be adjusted by the sphinx configuration option
+        kernel_doc_known_attrs
+
```

Comment:
can you add this documentation also in https://github.com/return42/linuxdoc/blob/master/docs/linuxdoc-howto/kernel-doc-directive.rst and examples to https://github.com/return42/linuxdoc/blob/master/docs/linuxdoc-howto/kernel-doc-tests.rst#exported-symbols

Reply:
done